laceworksdk/api/v2/activities.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework Activities API wrapper.
+"""
+
+from laceworksdk.api.search_endpoint import SearchEndpoint
+
+
+class ActivitiesAPI:
```

Comment:
classes missing docstrings

Reply:
Added class docstrings for newly implemented classes